game/scripts/vscripts/Sections/Chapter4/SectionWards.ts

```diff
@@ -18,6 +18,7 @@ const requiredState: RequiredState = {
 
 const markerLocation = Vector(-2200, 3800, 256);
 const wardLocation = Vector(-3400, 3800);
+const wardLocationSentry = Vector(-3400, 4000);
```

Comment:
Probably want to spawn the sentry ward there too (assuming it still uses wardLocation?)

Reply:
it does, will change